tools/@aws-cdk/prlint/lint.ts

```diff
@@ -575,6 +622,39 @@ export class PullRequestLinter {
       ],
     });
 
+    const codecovResults = {
+      patch: await this.codecovPatchSucceeded(sha),
+      patchPackages: await this.codecovPatchPackagesAwsCdkSucceeded(sha),
+      project: await this.codecovProjectSucceeded(sha),
+      projectPackages: await this.codecovProjectPackagesAwsCdkSucceeded(sha),
+    };
+
+    validationCollector.validateRuleSet({
```

Comment:
this will always fail at the start because the codecov reports take time to be generated. not sure how bad that is yet.